dockertest/environment.py

```diff
@@ -47,12 +55,7 @@ def get_selinux_context(path):
     When selinux is enabled, return the context of ``path``
     :param path: Full or relative path to a file or directory
     :return: SELinux context as a string or None if not set
+    :raises IOError: When file does not exist.
```

Comment:
If I see `raises` in the doc string, I expect to see `raises` in the code. I also wonder if there are other circumstances in which selinux might raise IOError, not just ENOENT, making the comment misleading. This isn't a critical issue, just not something I'd like to see as a habit.